test/functional/environment.js

```diff
@@ -4,27 +4,29 @@ const shell = require('shelljs');
 const getPort = require('get-port');
 
 function startEnvironment(serverPort) {
-	mockServer.start_mockserver({
+	return mockServer.start_mockserver({
 		serverPort,
 		trace: true
 	});
 }
 
 function stopEnvironment(serverPort) {
-	mockServer.stop_mockserver({
+	return mockServer.stop_mockserver({
 		serverPort
 	});
 }
 
 async function runTests() {
+	const ignoreArgs = ['--start-environment','--stop-environment']
+	const extraOptions = process.argv.slice(2).filter(arg=> !ignoreArgs.includes(arg));
```

Comment:
Do we need do to the args filtering here? Were the `--start-environment` and `--stop-environment` flags removed?

Reply:
i was still getting these args when running the jenkins job even though i removed them from package.json. when they're not removed they get passed to jest and it crashes saying that the args aren't recognized

Reply:
Might be a good idea to find out where those flags are added. Maybe they are part of the process and we should not change the logic